### ISSUE TITLE
Support Digital Ocean

### ISSUE THREAD

**Issue by BronsonQuick** [Chassis/Chassis#58]
I've made some progress on this using: https://github.com/smdahlen/vagrant-digitalocean  but I think I'll work on this as we work through https://github.com/Chassis/Chassis/issues/57

**Comment by pdennis33** [comment]
Hey @BronsonQuick,
First of all Chassis seems awesome and I got it to work on VirtualBox and Windows 7 x64, but I'm having all kinds of trouble trying to get it to work on DigitalOcean.  My plan is to have VirtualBox for local dev and DigitalOcean for production.  I tried this plugin but when I use the `vagrant up --provider=chassis_digitalocean` code it gives me an unauthorized error.  I regenerated a new token just in case from DO but still no luck.  I had tried the vagrant-digitalocean plugin first (before I found this one) and it would successfully create a droplet, but I couldn't work my way through all of the directory errors (directory not found, user not found etc).

Any tips on how I can get this to work?  I've been searching everywhere trying to get this setup to work.

Thank you!
Paul
